### ISSUE TITLE
FIA: Allow for retrieval of future time series

### ISSUE THREAD

**Issue by markushofmeister** [cambridge-cares/TheWorldAvatar#628]
Currently, FIA only retrieves historical time series data; however, it would be great to also visualise forecasts. I tested retrieving future time series using the settings below and it works; however, I have the impression that this simply retrieves all time series data available as it also shows quite a history.

```
{
    "class": "https://www.theworldavatar.com/kg/ontoems/ReportingStation",
    "metaFile": "ReportingStations.sparql",
    "timeFile": "ReportingStations_time.sparql",
    "databaseName": "postgres",
    "timeLimit": -48
}
```

Maybe we could change "timeLimit" into 2 key words to specify both the historic and future cut-off. I would hope that this is rather straight forward.

Furthermore, what happens if a Feature (i.e. ReportingStation) has both historic measurements and forecasts associated with it (which ideally should be visualised with different time limits)? Maybe we should extend the current time series sparql queries (i.e. "Measurement", "Name", "Unit") by "Forecast" and even provide those 2 time cut-off separately for measurements and forecasts in the fia-config.

**Comment by sm453** [comment]
I agree that this is needed, in particular separate time limits for historic and future time series.
